pylint_odoo/checkers/no_modules.py

```diff
@@ -203,10 +211,21 @@ class NoModuleChecker(BaseChecker):
             'help': 'List of methods where call to `super` is required.' +
                     'separated by a comma.'
         }),
+        ('invalid_commit', {
+            'type': 'string',
+            'metavar': '<string>',
+            'help': 'Dangerous method where call to `cr`.'
+        }),
     )
 
-    @utils.check_messages('translation-field',)
+    @utils.check_messages('translation-field',
+                          'invalid-commit')
     def visit_callfunc(self, node):
+        # Check cr.commit()
+        if "cr.commit" in node.as_string():
+            self.add_message('invalid-commit',
```

Comment:
That code trigger a syntax error, or a global name 'cr' is not defined, etc.

Also, the `as_string()` is to convert the node as a string and needs another string to allow the comparisson.